EMS/core-bundle/src/DataTable/Type/Revision/RemovedRevisionsDataTableType.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\DataTable\Type\Revision;
+
+use EMS\CoreBundle\Core\DataTable\Type\AbstractEntityTableType;
+use EMS\CoreBundle\Core\Revision\RemovedRevisionsService;
+use EMS\CoreBundle\Entity\ContentType;
+use EMS\CoreBundle\Form\Data\Condition\InMyCircles;
+use EMS\CoreBundle\Form\Data\DatetimeTableColumn;
+use EMS\CoreBundle\Form\Data\EntityTable;
+use EMS\CoreBundle\Form\Data\UserTableColumn;
+use EMS\CoreBundle\Roles;
+use EMS\CoreBundle\Routes;
+use EMS\CoreBundle\Service\ContentTypeService;
+use EMS\CoreBundle\Service\UserService;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RemovedRevisionsDataTableType extends AbstractEntityTableType
```

Comment:
I dislike the name of this class. What is about DeletedRevisionDataTableType

Reply:
Agree